x-pack/plugins/siem/common/endpoint/generate_data.test.ts

```diff
@@ -106,7 +113,11 @@ describe('data generator', () => {
         generations,
         percentTerminated: 100,
         percentWithRelated: 100,
-        relatedEvents: 4,
+        relatedEvents: [
+          { category: RelatedEventCategory.Driver, count: 1 },
```

Comment:
This means that the related events generated in this tree will be: 1 driver, 2 file, and 1 network type of events.